src/views/Home/components/Footer/index.view.tsx

```diff
@@ -1,49 +1,59 @@
 import * as React from "react";
+import { useState } from "react";
+import { ReactComponent as Arrow } from "images/arrow.svg";
+import { footerProps } from "views/Home/props/footer";
 import "./footer.scss";
 
 const FooterComponent: React.FC = () => {
+  const [showScroll, setShowScroll] = useState(false);
+  const checkScrollTop = () => {
+    if (!showScroll && window.pageYOffset > 400) {
+      setShowScroll(true);
+    } else if (showScroll && window.pageYOffset <= 400) {
+      setShowScroll(false);
+    }
+  };
+  window.addEventListener("scroll", checkScrollTop);
+
+  const scrollTop = () => {
+    window.scrollTo({ top: 0, behavior: "smooth" });
+  };
+
   return (
     <div className="Footer">
       <div className="Footer__images">
-        <div>Team!</div>
-        <div>Links!</div>
-        <div>Scroll!</div>
+        <div className="Footer__team">Team</div>
+        <div className="Footer__links">
+          {footerProps.map((prop) => (
```

Comment:
We can be really cool and destructure the `props`, thus eliminating the need to do `props.attribute`